packages/nuxt/src/app/components/client-only.mjs

```diff
@@ -37,7 +38,7 @@ export function createClientOnly (component) {
           ? createElementVNode(res.type, res.props, res.children, res.patchFlag, res.dynamicProps, res.shapeFlag)
           : h(res)
       } else {
-        return h('div', ctx.$attrs ?? ctx._.attrs)
+        return h('div', mergeProps(ctx.$attrs ?? ctx._.attrs, { key: randomUUID() }))
```

Comment:
do we need this to be random, or is assigning any key good enough?

Reply:
I think it should work too, i'll try